api/src/main/java/org/apache/iceberg/Metrics.java

```diff
@@ -190,6 +190,7 @@ private void readObject(ObjectInputStream in) throws IOException, ClassNotFoundE
     upperBounds = readByteBufferMap(in);
   }
 
+  @SuppressWarnings("DangerousJavaDeserialization")
```

Comment:
can you please add the same suppression to the method right above this method? It seems like it would be affected by this too